index.bs

```diff
@@ -385,8 +385,10 @@ Deleting a cookie by changing the expiry date:
 ```js
 try {
   const one_day_ms = 24 * 60 * 60 * 1000;
```

Comment:
Not new: maybe we should use const declaration in all of the examples where we're currently using this calculation inline.

Reply:
I think this is the only place this calculation is being used

Reply:
` expires: Date.now() - 24 * 60 * 60 * 1000 });`  up above?

Reply:
Ooh, I missed that. Thanks for the clarification!